ms-common-impl/src/main/java/net/trajano/ms/vertx/beans/TokenGenerator.java

```diff
@@ -1,10 +1,14 @@
 package net.trajano.ms.vertx.beans;
 
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
 import java.util.Random;
 
 import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.beans.factory.annotation.Autowired'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)